coretex/coretex/dataset/sequence_dataset/sequence_dataset.py

```diff
@@ -30,9 +30,33 @@ class SequenceDataset(BaseSequenceDataset, NetworkDataset[SequenceSample]):
         samples contain sequence data (.fasta, .fastq)
     """
 
+    metadata: CustomSample
+    pairedEnd: bool
```

Comment:
let's not store this into a variable, let's do either a function which returns this value `isPairedEnd()` or do `@property`